### ISSUE TITLE
landing page

### ISSUE THREAD

**Issue by timo-b-roettger** [troettge/RN-Norway#3]
- [x] agree on name
- [x] agree on domain
- [x] set up landing page
- [x] publish (among us)
- [x] get feedback on design
- [x] advertize

**Comment by agboch** [comment]
I vote for NORRN and norrn.no

**Comment by dsquintana** [comment]
I also like NORRN and norrn.no

**Comment by julienmayor** [comment]
I like NORRN and norrn.no, too. 
Alternatively, would RN-Norway work, or too long?

**Comment by troettge** [comment]
@dsquintana @julienmayor @agboch norrn.no secured. Please upload a picture of yourself that you would like on the webpage in square or round format.

**Comment by dsquintana** [comment]
Here you go ![quintana_profile](https://user-images.githubusercontent.com/8218312/140733109-b6cd0de8-475e-4489-a972-8254e94652f1.png)

**Comment by julienmayor** [comment]
![juju (2 of 1)_online-1](https://user-images.githubusercontent.com/60000307/140735464-6a778101-6877-44fb-a794-fb176350d17b.jpg)

**Comment by agboch** [comment]
Here is mine:
![DSC_1210](https://user-images.githubusercontent.com/48962315/140749632-8da3887a-8d77-4ed6-a1d3-c9ba6d730df8.jpg)

**Comment by troettge** [comment]
Hey team @dsquintana @agboch @julienmayor here is a view-only [link](https://preview.webflow.com/preview/norrn?utm_medium=preview_link&utm_source=designer&[…]n&preview=9f0ed50f643e93ade2b7fc4cf77429e8&workflow=preview) to the website. 

Please have a careful look at it, maybe use different browsers/devices and make a list here of things that should be fixed or discussed. Please always give a short reason why if it is not an obvious disfunctionality.

- [ ] change pink highlight from mission statement to black because looks like link

**Comment by julienmayor** [comment]
This looks great!

Some very minor comments:
- In "terms of reference", the "More information below" is in another font/level. Any reason for this? It looks a bit like it is temporary (which may be the point!).
- In "A potential part-time administrator  might support it in the future as NORRN coordinator(assuming funds can be secured for this).", I believe there are two spaces after "administrator", and none after "coordinator".
- In "Implementation", I'd add spaces before each point in the first paragraph.
- In "Local node...". In the bullet point "Initiating or coordinating a Reproducibility Journal Club (e.g. https://reproducibilitea.org/)", do we want to have an hyperlink there?
- In the Code of conduct, I'd add a line before the second, and before the third paragraphs - in line with other sections.
- Also there, when talking about "initiating members" in the code of conduct - do we want to create hyperlinks to the Members section?
- Under "Members" - first sentence, I think there is an extra space before "our colleagues" and missing text after "read our ...".
- I haven't tried the contact form and the email contact (mail@business.com??). Not sure if the functionality works as of now.

Great job!

**Comment by dsquintana** [comment]
This looks great! Seems like the contact form only works when the page is live (I tried sending a message). But if the form points to norrepro@gmail.com then I'm sure it will be fine. Tried this on chrome, safari, and mobile safari

**Comment by sirileknes** [comment]
![Siri 02](https://user-images.githubusercontent.com/93480384/143263922-4750fa35-2844-4f17-96e9-448747af551f.jpg)

**Comment by troettge** [comment]
> This looks great! Seems like the contact form only works when the page is live (I tried sending a message). But if the form points to [norrepro@gmail.com](mailto:norrepro@gmail.com) then I'm sure it will be fine. Tried this on chrome, safari, and mobile safari

Yeah I think it needs to be live. I have connected it with our email, so this is the one thing we can test as soon as its live before we advertise.

**Comment by troettge** [comment]
@julienmayor @sirileknes @dsquintana @agboch 

[NORRN](https://www.norrn.no/) is online, there are already a bunch of things I noticed. Just add things you noticed here as tick boxes:

- [ ] need to change tab icon to logo
- [ ]

**Comment by julienmayor** [comment]
Great work, Timo!
I've tried it out on a Mac with Google Chrome and (old) iPhone with Google Chrome. Both work well!
I've added a few bullet points above - otherwise this looks good!

**Comment by agboch** [comment]
Looks great! Tested on new iPhone with Google Chrome and Safari and on a Windows machine on Google Chrome and Firefox. All works well! Sent a few test contact inquires, including one anonymous one. 
One question - since everything is there in English, should we also write University of Oslo (and not Universitetet i Oslo) in our afiliations?

**Comment by dsquintana** [comment]
Looks good from my end. I can confirm that the submission form enquiries are all coming through

**Comment by sirileknes** [comment]
I think the website looks FANTASTIC! Well done, Timo especially!
For the terms of reference, I wondered if this statement is really correct for us:
NORRN originated from activities seeking to understand the factors that contribute to poor research reproducibility and replicability, and to promote possible approaches to counter these and improve the quality and robustness of the research we produce.

In my view we didn't originate from activities seeking to understand, but instead primarily to promote good research, so perhaps we could rewrite as "NORRN originated from activities seeking to promote possible approaches to counter poor reproducibility in research and to improve the quality and robustness of the research we produce." or something like that?

**Comment by troettge** [comment]
I have added a reCAPTCHA function to avoid too many spam mails (we have a monthly limit of 1000 messages)

**Comment by troettge** [comment]
:partying_face: Code of Conduct and Terms of Reference has been updated, everything seems to work so we are good to go folks :partying_face: 

@dsquintana @sirileknes @agboch @julienmayor 

Let the advertising begin. Do we need a strategy? My reasoning would be:
1. Tweet with new twitter account and tag all repro network twitter accounts
2. Retweet from our personal accounts

What say you, @dsquintana?
